server/src/main/java/org/apache/cassandra/sidecar/routes/AbstractHandler.java

```diff
@@ -331,4 +334,34 @@ public static String extractHostAddressWithoutPort(String address)
         }
         return address;
     }
+
+    protected StorageOperations getStorageOperations(String host)
+    {
+        CassandraAdapterDelegate delegate = this.metadataFetcher.delegate(host);
+        StorageOperations storageOperations = delegate == null ? null : delegate.storageOperations();
+        if (storageOperations == null)
+        {
+            throw cassandraServiceUnavailable();
```

Comment:
Do we want to add some text to this error?

Reply:
cassandraServiceUnavailable currently says "Cassandra service is unavailable". The same message goes back in client's response. As this is an internal error, adding further info may leak internal info to the client. We use the same cassandraServiceUnavailable almost everywhere without adding further details, must be the same reason I am assuming. 

Any other kind of exception is caught in AbstractHandler's handle() and logged and updated in the response.